### PR TITLE
Update Menu Icon

### PR DESCRIPTION
**Pull request by chiaramooney**
Update menu icon to hamburger menu for cleaner look and adjust page layout to fit new dimensions of menu button.

image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -35,7 +47,13 @@ function RNGalleryScreenWrapper({navigation}) {
   const Component = RNGalleryList[state.index].component;
   return (
     <View style={styles.container}>
-      <Button title="Menu" onPress={() => navigation.openDrawer()} />
+      <TouchableHighlight
+        style={styles.menu}
+        onPress={() => navigation.openDrawer()}
+        activeOpacity={0.5783}
```
Comment (chrisglein) [inline src/App.tsx:53]: Why opacity? Should this be baked into the color? And should it relate to some defined `PlatformColor`?

File: src/App.tsx [hunk 2]
```diff
@@ -35,7 +47,13 @@ function RNGalleryScreenWrapper({navigation}) {
...
+        underlayColor="rgba(0, 0, 0, 0.0241);">
```
Comment (chrisglein) [inline src/App.tsx:54]: Similar question: Where does this magic opacity number come from?

File: src/App.tsx (hunk 1 above)
Comment (chiaramooney) [reply src/App.tsx:53]: Sync'd offline

File: src/App.tsx (hunk 2 above)
Comment (chiaramooney) [reply src/App.tsx:54]: Sync'd offline

**Review by chrisglein — APPROVED** [review]

**Pull request merged**